src/trivyHelper.ts

```diff
@@ -35,6 +37,65 @@ export async function getTrivy(): Promise<string> {
     return trivyToolPath;
 }
 
+export function getText(trivyStatus: number): string {
+    const vulnerabilityIds = getVulnerabilityIds(trivyStatus);
+    return `**CVE's** -\n${vulnerabilityIds.join('\n')}`;
+}
+
+export function getSummary(trivyStatus: number): string {
+    let summary = '';
+    switch(trivyStatus) {
+        case 0:
+            summary = 'No vulnerabilities were detected in the container image'
+            break;
+        case TRIVY_EXIT_CODE:
+            const vulnerabilities = getVulnerabilities();
+            const total = vulnerabilities.length;
+            const unknownCount = vulnerabilities.filter(v => v['Severity'].toUpperCase() === 'UNKNOWN').length;
+            const lowCount = vulnerabilities.filter(v => v['Severity'].toUpperCase() === 'LOW').length;
+            const mediumCount = vulnerabilities.filter(v => v['Severity'].toUpperCase() === 'MEDIUM').length;
+            const highCount = vulnerabilities.filter(v => v['Severity'].toUpperCase() === 'HIGH').length;
+            const criticalCount = vulnerabilities.filter(v => v['Severity'].toUpperCase() === 'CRITICAL').length;
+        
+            summary = `Found ${total} vulnerabilities -\nUNKNOWN: ${unknownCount}\nLOW: ${lowCount}\nMEDIUM: ${mediumCount}\nHIGH: ${highCount}\nCRITICAL: ${criticalCount}`;
+            break;
+        default:
+            summary = 'An error occured while scanning the container image for vulnerabilities';
+            break;
+    }
+    
+    return summary;
+}
+
+function getVulnerabilityIds(trivyStatus: number): string[] {
+    let vulnerabilityIds: string[] = [];
+    if(trivyStatus == TRIVY_EXIT_CODE) {
+        const vulnerabilities = getVulnerabilities();
+        vulnerabilityIds = vulnerabilities.map(v => v['VulnerabilityID']);
+    }
+
+    return vulnerabilityIds;
+}
+
+function getTrivyOutput(): any {
+    const path = fileHelper.getTrivyOutputPath();
+    return fileHelper.getFileJson(path);
+}
+
+function getVulnerabilities(): any[] {
+    const trivyOutputJson = getTrivyOutput();
+    let vulnerabilities: any[] = [];
+    trivyOutputJson.forEach((ele: any) => {
+        if(ele && ele["Vulnerabilities"]) {
+            ele["Vulnerabilities"].forEach((cve: any) => {
+                vulnerabilities.push(cve);
+            });
+        }
```

Comment:
Don't you want to cluster them based on severity?

Reply:
Filed issue #7 to track the changes in check run text.